indico/modules/events/contributions/templates/display/user_contribution_list.html

```diff
@@ -1,44 +1,17 @@
 {% extends 'events/display/conference/base.html' %}
+{% from 'events/contributions/display/_user_contribution_list.html' import render_contribution_section %}
 
 {% block title -%}
     {{- page_title -}}
 {%- endblock %}
 
 {% block content -%}
-    <section>
-        <div class="i-box no-padding">
-            <div class="i-box-content">
-                <ul class="group-list no-content-before with-buttons">
-                    {% for contrib in contributions %}
-                        <li class="flexrow f-j-space-between">
-                            <span class="text">
-                                <a class="js-mathjax" href="{{ url_for('.display_contribution', contrib) }}">
-                                    {{- contrib.title -}}
-                                </a>
-                                {% if contrib.code -%}
-                                    <span class="contrib-code">
-                                        ({{ contrib.code }})
-                                    </span>
-                                {%- endif %}
-                            </span>
-                            <div class="group">
-                                {% if contrib.can_edit(session.user) -%}
-                                    <a href="#" class="icon-edit js-edit-contribution"
-                                       title="{% trans %}Edit this contribution{% endtrans %}"
-                                       data-title="{% trans title=contrib.title %}Edit contribution '{{ title }}'{% endtrans %}"
-                                       data-href="{{ url_for('.manage_update_contrib', contrib, standalone=true) }}"
-                                       data-ajax-dialog
-                                       data-reload-after></a>
-                                {%- endif %}
-                            </div>
-                        </li>
-                    {% else %}
-                        <li>{% trans %}You don't have any contributions at the moment.{% endtrans %}</li>
-                    {%- endfor %}
-                </ul>
-            </div>
-        </div>
-    </section>
+    {{ render_contribution_section(contributions['primary'], title=gettext('Primary author'),
+                                   no_contribs_msg=gettext('You are not the primary author of any contribution.')) }}
+    {{ render_contribution_section(contributions['secondary'], title=gettext('Co-author'),
+                                   no_contribs_msg=gettext('You are not the co-author of any contribution.')) }}
+    {{ render_contribution_section(contributions['speaker'], title=gettext('Speaker'),
+                                   no_contribs_msg=gettext('You are not the speaker of any contribution.')) }}
```

Comment:
The `_('...')` alias exists in templates as well, and I'd use `contributions.primary` etc (more common in templates instead of using the explicit dict item notation)